src/v/cluster/id_allocator_frontend.cc

```diff
@@ -29,6 +29,15 @@
 namespace cluster {
 using namespace std::chrono_literals;
 
+static ss::future<bool> sleep_abortable(std::chrono::milliseconds dur) {
+    try {
+        co_await ss::sleep_abortable(dur);
```

Comment:
isn't there a `seastar::sleep_abortable()` don't you need to pass the `seastar::abort_source&` to this anyway?

Reply:
[There](http://docs.seastar.io/master/sleep_8hh.html) are several overloads not all of them require abort sources. I've introduced a helper function to wrap the invocation with try/catch to intercept sleep_aborted exception.